test/unit/plugins/listr-karma.test.js

```diff
@@ -98,7 +81,7 @@ describe('SpecReporter', function () {
 							failedSpec = {
 								suite: ['suite name'],
 								description: 'description of test',
-								log: ['log', 'test/add.test.js']
+								log: ['2+2 doest not equal carrot', 'test/add.test.js:1:12']
```

Comment:
`2+2 doest not equal carrot` 😂

Reply:
Hey now, I'm almost certain that it does not equal carrot :P